Bio/Graphics/BasicChromosome.py

```diff
@@ -486,7 +486,7 @@ def _draw_label(self, cur_drawing):
 
 
 def _spring_layout(desired, minimum, maximum, gap=0):
-    """Try to layout label co-ordinates (or other floats, PRIVATE).
+    """Try to layout label co-ordinates or other floats, (PRIVATE).
```

Comment:
The original was fine, but if you want to have the exact string ``(PRIVATE)`` please remove the comma:

```
    """Try to layout label co-ordinates or other floats (PRIVATE).
```